test/integ/quote-classic.test.ts

```diff
@@ -481,6 +482,7 @@ describe('quote', function () {
     [ChainId.ZORA_SEPOLIA]: null,
     [ChainId.ROOTSTOCK]: null,
     [ChainId.BLAST]: WNATIVE_ON(ChainId.BLAST),
+    [ChainId.ZORA]: WNATIVE_ON(ChainId.ZORA),
```

Comment:
Looks like these is a duplicate ZORA entry on like 480/481

Reply:
Good catch. In-place updated now.